values/values.go

```diff
@@ -148,6 +151,16 @@ func (v value) Equal(r Value) bool {
 	}
 }
 
+func (v value) Retain() {
+	n := New(v.v)
+	n.Retain()
+}
+
+func (v value) Release() {
+	n := New(v.v)
+	n.Release()
+}
```

Comment:
`make checkdocs` seems to be complaining about these implementations of retain and release. Not sure why this would cause a panic, but I will investigate on Monday.

Reply:
`New` will return a `*value`/`Value` so the `Release/Retain` called will be the same methods that are defined here, ie it will just recurse into itself. I think you need to do a manual `switch` here and dispatch.